index.d.ts

```diff
@@ -246,31 +253,35 @@ declare const captureWebsite: {
 	import captureWebsite = require('capture-website');
 
 	(async () => {
-		await captureWebsite.file('https://sindresorhus.com', 'screenshot.png');
+		await captureWebsite.file('https://sindresorhus.com', 'screenshot-url.png');
+
+		await captureWebsite.file('<h1>Awesome!</h1>', 'screenshot-html.png', {
+			inputType: 'html'
+		});
 	})();
 	```
 	*/
 	file(
-		url: string,
+		input: string,
 		outputFilePath: string,
 		options?: captureWebsite.FileOptions
 	): Promise<void>;
 
 	/**
-	Capture a screnshot of the given `url`.
+	Capture a screenshot of the given `input`.
 
-	@param url - The URL, file URL, data URL, or local file path to the website.
+	@param input - The URL, file URL, data URL, or local file path to the website, or HTML content of the website.
```

Comment:
You need to update this and the other sentence too. I thought that was implied when I commented on the first one.

Reply:
Double checked.